### ISSUE TITLE
Double quotes not available in DB browser for Sqlite on Debian Buster (French installation)

### ISSUE THREAD

**Issue by manu7060** [sqlitebrowser/sqlitebrowser#2432]
Details for the issue
--------------------

#### What did you do?
Modifying a table structure or a schema.  This is a very basic action, for example adding a column to an existing table.

#### What did you expect to see?
I expected it changes the table and columns.

#### What did you see instead?
IT returns an error and it is impossible to change the table and see the new column. Inspecting the SQL string, I noticed it use single quotes while the Windows and the Mac OS versions use double quotes. In this version of DB Browser there is no option to change the type of quotes.  I am convinced this is the cause of errors.
The DB Browser is unusable on this computer.

Useful extra information
-------------------------
> DB4S v3.10 as available in the Debian repository
> Debian Buster, French installation.

**Comment by chrisjlocke** [comment]
v3.12 would be the latest version.  Have you tried compiling it yourself?

**Comment by justinclift** [comment]
As @chrisjlocke mentions, the version of the program you're using is several major releases out of date.

We have Ubuntu packages available, which **might** work for Debian too:

&nbsp; &nbsp; https://github.com/sqlitebrowser/sqlitebrowser/#stable-release-1

Maybe give those a go? :smile:

**Comment by manu7060** [comment]
The latest version of Sqlite Browser in Debian repository is 3.10.1. will an updated version be available soon?  I would like to use it for student computers, so it must be easy to deploy.

**Comment by lucydodo** [comment]
We do not have permission to manage Debian repositories. However, there is a repository managed by our project member.
You can add that and install the latest version by installing it with the following command.

```
$ sudo add-apt-repository -y ppa:linuxgndu/sqlitebrowser
$ sudo apt-get update    # Automatically executed in the latest debian build (only if not executed)
$ sudo apt-get install sqlitebrowser
```

**Comment by manu7060** [comment]
OK, now a get this message: NO_PUBKEY B0C3F48A7F2583EA
And it still install the old version from the Debian repository

**Comment by lucydodo** [comment]
Sorry. I knew that the Ubuntu PPA can work on Debian too, tested it on the my VM and it wasn't.
I will find out another solution. 🤔

**Comment by manu7060** [comment]
@lucydodo Yes, if you can build a Debian version it could help me a lot. Thank you in advance.  So I can complete the package for the student workstations.
